runtime/string.cpp

```diff
@@ -0,0 +1,10 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2021 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#include "runtime/kphp_core.h"
+
+// Don't move this destructor to the headers, it spoils addr2line traces
+string::~string() noexcept {
+  destroy();
```

Comment:
Why mixed::destroy() was moved to a .cpp file, but string::destroy() wasn't?

Reply:
string::destroy() is quite simple and it is used in string.inl.

mixed::destroy() is used in in mixed.inl as well, but it produces much more asm (due to array destructor), therefore I moved it here.